### PR TITLE
feat: Add n1.medium instancetype

### PR DESCRIPTION
**Pull request by 0xFelix**
**What this PR does / why we need it**:

Add the n1.medium instancetype which still provides 4 guest cpus but only 4Gi of memory.

**Which issue(s) this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close the issue(s) when PR gets merged)*:
Fixes #

**Special notes for your reviewer**:

**Release note**:

```release-note
Added the n1.medium instancetype which still provides 4 guest cpus but only 4Gi of memory
```

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -241,6 +241,7 @@ n1.2xlarge  |  16  |  32Gi
 n1.4xlarge  |  32  |  64Gi
 n1.8xlarge  |  64  |  128Gi
 n1.large  |  4  |  8Gi
+n1.medium  |  4  |  4Gi
```
Comment (orelmisan) [inline README.md:244]: nit: Please consider moving it before `n1.large`.

**Review by orelmisan — APPROVED** [review]
Thank you @0xFelix.
The definition for the memory to be allocated by the host's hugepages comes from a preference, right?

**Comment by jcanocan** [comment]
Thanks @0xFelix!
/lgtm

File: README.md (hunk 1 above)
Comment (0xFelix) [reply README.md:244]: The list is auto-generated and sorted alphabetically. :)

**Pull request merged**